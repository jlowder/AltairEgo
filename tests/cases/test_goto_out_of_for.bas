10 PRINT "TEST: GOTO OUT OF FOR LOOP"
20 FOR I = 1 TO 5
30 PRINT "IN LOOP, I="; I
40 IF I = 3 THEN GOTO 70
50 NEXT I
60 GOTO 80
70 PRINT "JUMPED OUT OF LOOP"
80 FOR I = 1 TO 2
90 PRINT "SECOND LOOP, I="; I
100 NEXT I
110 PRINT "TEST PASSED"
120 END
