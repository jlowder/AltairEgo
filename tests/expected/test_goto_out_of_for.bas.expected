Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
TEST: GOTO OUT OF FOR LOOP
IN LOOP, I= 1
IN LOOP, I= 2
IN LOOP, I= 3
JUMPED OUT OF LOOP
SECOND LOOP, I= 1
SECOND LOOP, I= 2
TEST PASSED
OK
